lib/lambdas/ssmPublisher/handler.ts

```diff
@@ -0,0 +1,65 @@
+import * as lambda from 'aws-lambda';
+import * as log from 'lambda-log';
+import { SsmGateway } from './ssm-param-gateway';
+
+/** Exports here are used by the cdk infrastructure */
+export const SLEEP_SECONDS = 'SLEEP_SECONDS';
+
+// const PHYSICAL_ID = '6db73885-e1cf-4510-a926-31d058538016';
+/**
+
+/**
+ * Entrypoint for the cloudformation custom resource.
+ */
+export async function handler(event: lambda.CloudFormationCustomResourceEvent): Promise<string> {
+
+    log.options.debug = true;
+    log.info('event', { event, envVars: process.env });
+
+    if (!event.RequestType) {
+        const err = `event.RequestType not found. Did you mean to use the custom resource handler?`
+        throw new Error(err);
+    }
+
+    if (!process.env[SLEEP_SECONDS]) throw Error("SLEEP_SECONDS parameter not found.")
+
+    try {
+        // uncomment to spread out the load on SSM a little further.
```

Comment:
Is the intent still to use the sleep command? Maybe you could make it optional, and only sleep if there is a value passed in, rather than erroring if there is no value.

Reply:
This code was gutted already